paasta_tools/cli/cmds/spark_run.py

```diff
@@ -983,7 +984,9 @@ def paasta_spark_run(args):
     app_base_name = get_spark_app_name(args.cmd or instance_config.get_cmd())
 
     if args.enable_compact_bin_packing:
-        document = POD_TEMPLATE.format(spark_app_name=app_base_name)
+        document = POD_TEMPLATE.format(
+            spark_pod_label=limit_size_with_hash(f"exec-{app_base_name}"),
```

Comment:
oh, one other thing - do we need to sanitize this (i.e., call `sanitise_kubernetes_name()` before `limit_size_with_hash`)?

i guess that if we haven't run into any issues yet, it's probably not required - but might save us some trouble in the future if anyone tries to use a spark app name with a disallowed character :p

Reply:
It hasn't given any issue so far apparently, but that's a good point. I think since it's only a label it can be anything, except being longer than 63 char. Out of excess of precaution, I'm going to apply that function here as well!